react/IconSocial.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { createElement } from 'react'
+import PropTypes from 'prop-types'
+import Icon from './components/Icon'
```

Comment:
separate relative imports from static ones
```suggestion

import Icon from './components/Icon'
```